tasks/pkg-debian/install-apm-inject-latest.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install the APM inject library
```

Comment:
In order to make the installation more performant, can we use just one task to install multiple packages? Doing one APT invocation saves a lot of time in the run when compared to multiple ones (same applies for the RPM installation).

The `name` argument accepts a list of packages, a loop shouldn't be necessary.